guides/common/assembly_using-external-databases.adoc

```diff
@@ -15,7 +15,7 @@ endif::[]
 To create and use external databases for {Project}, you must complete the following procedures:
 
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 8 server to host the external databases.
+Prepare the {EL} to host the external databases.
```

Comment:
How about leaving out the specific OS altogether? It's aimed to provide an overview; the details can be in the chapter.
```suggestion
Prepare a host for the external databases.
```